src/main/java/com/puppycrawl/tools/checkstyle/meta/JavadocMetadataScraper.java

```diff
@@ -218,7 +224,12 @@ public void finishJavadocTree(DetailNode rootAst) {
             }
             else {
                 try {
-                    XmlMetaWriter.write(moduleDetails);
+                    if (moduleDetails.getDescription().isEmpty()) {
```

Comment:
Can't this be moved outside the try/catch to show which is clearly throwing the exception which should be only the write method.
````
if (test) {
  log();
}
else {
  try {...} catch {...}
]
````

Reply:
done